Content.Shared/Prying/Systems/PryingSystem.cs

```diff
@@ -71,6 +74,10 @@ public bool TryPry(EntityUid target, EntityUid user, out DoAfterId? id, EntityUi
         if (!comp.Enabled)
             return false;
 
+        // DeltaV - Cancel if no power and needs power
+        if (TryComp<PryingUsePowerComponent>(tool, out var powerComp) && !_power.HasCharge(tool, powerComp.UseCost, null, user))
+                return false;
```

Comment:
remove, raise the event in CanPry on the tool as well so this component can handle it